app/models/policies.rb

```diff
@@ -9,6 +9,10 @@ def self.all
     POLICIES
   end
 
+  def self.options_for_select
+    all.collect { |c| [c.short_name, c.to_s.underscore] }
```

Comment:
We could maybe utilise `c.routing_name` here? It might make matching a bit easier as well